core/services/keeper/orm.go

```diff
@@ -168,7 +176,7 @@ func (korm ORM) CreateEthTransactionForUpkeep(sqlDB *sql.DB, upkeep UpkeepRegist
 	if etx.ID == 0 {
 		return etx, errors.New("a keeper eth_tx with insufficient eth is present, not creating a new eth_tx")
 	}
-	err = korm.DB.First(&etx).Error
+	err = tx.First(&etx).Error
```

Comment:
This sort of mistake might be best avoided by encapsulating this logic in one place

Reply:
I mean we'd be better off using `returning` and scan anyway, right?

Reply:
will defer to @RyanRHall's ticket addressing this code